### PR TITLE
Make ExecutionEnvironment a pointer

### PR DESCRIPTION
**Pull request by swapnilraj**

### CODE REVIEW

File: warp/yul/ToCairoVisitor.py [hunk 1]
```diff
@@ -380,10 +382,12 @@ def _make_external_function(
             params += "calldata_size, calldata_len, calldata : felt*"
             init_exec_env = (
                 f"let (returndata_ptr : felt*) = alloc()\n"
-                f"local exec_env : ExecutionEnvironment ="
-                f"ExecutionEnvironment(calldata_size=calldata_size,"
+                f"let (local exec_env : ExecutionEnvironment*) = alloc()\n"
```
Comment (murcake) [inline warp/yul/ToCairoVisitor.py:385]: I think this line can be removed

**Review by murcake — APPROVED** [review]

File: warp/yul/ToCairoVisitor.py (hunk 1 above)
Comment (JorikSchellekens) [reply warp/yul/ToCairoVisitor.py]: Removed
=

Comment (murcake) [reply warp/yul/ToCairoVisitor.py]: Probably will require regenerating files

**Comment by swapnilraj** [comment]
I have updated the test expectations, the test should pass now! 🤞

**Comment by murcake** [comment]
Let's squash the last two commits and merge

**Pull request merged**